examples/bank/bank.go

```diff
@@ -149,11 +146,13 @@ func (bank *Bank) continuousMoneyTransfer(cash int64) {
 			}
 			return runner.Run(batchWrite)
 		}
+		var err error
 		if *useTransaction {
-			if err := bank.db.Tx(func(tx *client.Tx) error { return transferMoney(tx) }); err != nil {
-				log.Fatal(err)
-			}
-		} else if err := transferMoney(bank.db); err != nil {
+			err = bank.db.Tx(func(tx *client.Tx) error { return transferMoney(tx) })
```

Comment:
`func (db *DB) Tx(retryable func(tx *Tx) error) error {`

Isn't `retryable` of the same type `func(tx *client.Tx) error`?